instance_config.go

```diff
@@ -156,6 +157,16 @@ func (config *InstanceConfig) Validate(registry Registry) error {
 		for _, api := range serverConfig.APIs {
 			presentApis[api.Binding()] = registry.Get(api.Binding())
 		}
+		for _, bp := range serverConfig.BindPoints {
+			ve := serverConfig.Identity.ValidFor(bp.Address)
+			if ve != nil {
+				errs = append(errs, ve)
+			}
+		}
+	}
+
+	if len(errs) > 0 {
+		return fmt.Errorf("%v", errs)
```

Comment:
Is there a reason to not use errors.Join()?

```
// Join returns an error that wraps the given errors.
// Any nil error values are discarded.
// Join returns nil if every value in errs is nil.
// The error formats as the concatenation of the strings obtained
// by calling the Error method of each element of errs, with a newline
// between each string.
//
// A non-nil error returned by Join implements the Unwrap() []error method.
func Join(errs ...error) error {
```

Does the same thing but preserves the individual errors.

Reply:
lol nope other than i tried to use it in a differnt PR but we had a different package reference ... updated!